packages/client/tests/functional/metrics/_matrix.ts

```diff
@@ -0,0 +1,17 @@
+import { defineMatrix } from '../_utils/defineMatrix'
+
+export default defineMatrix(() => [
+  [
+    {
+      provider: 'sqlite',
+    },
+  ],
+  [
+    {
+      engineType: 'library',
```

Comment:
I really like that we could test the engineTypes this way! However, the CI already does that by running all the tests with binary and again with library, and it does it via the `PRISMA_CLIENT_ENGINE_TYPE` env var and via `getClientEngineType` in the code. tl;dr; You don't need to add anything and can remove both engine types.